scripts/cmake/vcpkg_configure_cmake.cmake

```
@@ -87,6 +91,18 @@ function(vcpkg_configure_cmake)
         set(arg_LOGNAME config-${TARGET_TRIPLET})
     endif()
 
+    set(manually_specified_variables)
```

```suggestion
    set(manually_specified_variables "")
```

this is a new guideline.